README.md

```diff
@@ -1 +1,15 @@
 # hw01
+
+## **About me**
+Hi! My name is **Zhilan Fang**. I'm a MAPSS student with a concentration in Sociology. I'm from China and will stay in [UChicago Beijing Center](https://www.uchicago.cn/) during the fall quarter. I'm broadly interested in topics about *inequality, social media, and opinion formation*. I'm looking forward to learning more computational skills in this course and applying them to my studies.
+
+## **Other things you might want to know**
+* Hold a bachelor's degree in **Journalism** from [Tsinghua University](https://www.tsinghua.edu.cn/en/index.htm)
+* Love watching films and documentaries, and also had experience shooting a short documentary about **Tibetan folk songs**
+* Definitely a dog person
+* Also love cooking but is still trying to improve my cooking skills
+
+Here is a photo of me:
+
+![profile](https://github.com/zlfang97/hw01/blob/master/profile.jpg?raw=true)
```

Comment:
Because this picture is included in your repo, you can use a relative link like so: 
```
![profile](profile.jpg)
```